### PR TITLE
Project Structure

### PR DESCRIPTION
**Pull request by right-mouse**
Initial project structure and boilerplate. I opted for a simplistic approach that didn't involve defining a full blown asdf system since this project shouldn't have any external dependencies.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,9 @@
+*.abcl
+*.fasl
```
Comment (boarnoah) [inline .gitignore:2]: Are these from your own experience? I'm looking at the list here to get an idea re: typical lisp gitignore https://github.com/github/gitignore/blob/main/CommonLisp.gitignore

File: README.md [hunk 2]
```diff
@@ -1 +1,6 @@
-# advent-of-code-2015
\ No newline at end of file
+# Advent of Code 2015
+
+My attempt at https://adventofcode.com/2015 in Common Lisp.
+
+## Running
+TBD
```
Comment (boarnoah) [inline README.md]: I wouldn't mind this, even if its just for posterity

**Review by boarnoah — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (right-mouse) [reply .gitignore:2]: No I haven't come across these personally. It was just part of the boilerplate I got from using `cl-project` to init a new project.

https://lispcookbook.github.io/cl-cookbook/getting-started.html#creating-a-new-project

File: README.md (hunk 2 above)
Comment (right-mouse) [reply README.md]: Done.

**Review by boarnoah — APPROVED** [review]

**Pull request merged**